### DISCUSSION TITLE
We want to Contribute providing developer

### DISCUSSION THREAD

**Discussion by esafwan** [alphabit-technology/erpnext-restaurant#30] [General]
Hello,
We are currently working on a restaurant feature in association with ERPNext core team. Can we get on a web call to explore the possibility of collaborating? If it works we can commit developers for continued contribution to this branch. 

Do let me know a possible time to further discuss this. 

Thanks

**Comment by alphabit-technology** [comment]
Can you write me to alfredrz2012@gmail.com please.
